examples/job/get-file-from-job.py

```diff
@@ -0,0 +1,282 @@
+CLUSTER = "cluster-001"
+
+
+# ### Initialize endpoints
+
+# In[]:
+
+
+job_endpoints = JobEndpoints(*ENDPOINT_ARGS)
+project_endpoints = ProjectEndpoints(*ENDPOINT_ARGS)
+material_endpoints = MaterialEndpoints(*ENDPOINT_ARGS)
+raw_property_endpoints = RawPropertiesEndpoints(*ENDPOINT_ARGS)
+bank_workflow_endpoints = BankWorkflowEndpoints(*ENDPOINT_ARGS)
+
+
+# Next, we retrieve the owner and project IDs as they are needed by the endpoints. Account's default material is used to extract the owner ID. One can extract the owner ID from any other account's [entities](https://docs.exabyte.io/entities-general/overview/).
+
+# In[]:
+
+
+owner_id = material_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]["owner"]["_id"]
+project_id = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]["_id"]
+
+
+# ### Create workflow
+# 
+# Copy bank workflow (template) to the account's workflows collection.
+
+# In[]:
+
+
+workflow_id = bank_workflow_endpoints.copy(BANK_WORKFLOW_ID, owner_id)["_id"]
+
+
+# ### Import materials
+# 
+# Import materials from materials project with the above tags.
+
+# In[]:
+
+
+materials = material_endpoints.import_from_materialsproject(MATERIALS_PROJECT_API_KEY, MATERIALS_PROJECT_ID, owner_id)
+
+
+# Create a materials set and move the materials into it.
+
+# In[]:
+
+
+materials_set = material_endpoints.create_set({"name": MATERIALS_SET_NAME, "owner": {"_id": owner_id}})
+for material in materials: material_endpoints.move_to_set(material["_id"], "", materials_set["_id"])
+
+
+# ### Create jobs
+# 
+# Create jobs for the materials above.
+
+# In[]:
+
+
+compute = job_endpoints.get_compute(CLUSTER, PPN, NODES, QUEUE, TIME_LIMIT)
+jobs = job_endpoints.create_by_ids(materials, workflow_id, project_id, owner_id, JOB_NAME_PREFIX, compute)
+
+
+# Create a jobs set and move the jobs into it.
+
+# In[]:
+
+
+jobs_set = job_endpoints.create_set({"name": JOBS_SET_NAME, "projectId": project_id, "owner": {"_id": owner_id}})
+for job in jobs: job_endpoints.move_to_set(job["_id"], "", jobs_set["_id"])
+
+
+# ### Submit jobs
+# Submit the jobs for execution.
+
+# In[]:
+
+
+for job in jobs: job_endpoints.submit(job["_id"])
+
+
+# Monitor the jobs and print the status until they are all finished.
+
+# In[]:
+
+
+job_ids = [job["_id"] for job in jobs]
+wait_for_jobs_to_finish(job_endpoints, job_ids)
+
+
+# ### Retreive a list of job files
+# 
+# Here, we'll get a list of all files that belong to the job.
+
+# In[]:
+
+
+job = jobs[0]
+files = job_endpoints.list_files(job['_id'])
+filenames = [file['key'] for file in files]
+for filename in filenames:
+    print(filename)
+
+
+# # Get more information on the OUTCAR
+# The OUTCAR file is where VASP shows it work, and contains a great deal of useful information that you might want to parse programatically. Very commonly in computational chemistry, users will "tail" the OUTCAR (that is, only printing the last several lines) to check whether it has completed.
```

Comment:
Pycharm should enforce max line width for the code (127 char?). More than that is difficult to read